game/src/MusicConductor/SongInfo.gd

```diff
@@ -0,0 +1,11 @@
+extends Node
+class_name SongInfo
+
+var fullyQualifiedPath : String = ""
+var readableName : String = ""
+var loadedStream : Resource
+
+func _init(dirname:String, fname:String):
+		fullyQualifiedPath = dirname + fname
+		readableName = fname.replace(".mp3","").replace("_", " ")
```

Comment:
This prevents non-mp3s from being usable.

Reply:
Addressed by most recent commit